.github/workflows/markdownLint.yaml

```diff
@@ -0,0 +1,34 @@
+name: Markdown Lint
+
+on:
+  push:
+    branches:
+      - '**'
+    tags-ignore:
+      - '**'
+  pull_request_target:
+    branches:
+      - '**'
```

Comment:
Should we be running on every pull request, or should it be only those that target into `main` ? I wouldn't expect there would be many that aren't going into main

Reply:
I guess only for the PRs, the same as spell checker?